### PR TITLE
EE-6402: Jenkins Migration Phase 1 - 3.x: Migrate Grizzly Third Party Jobs

### PR DESCRIPTION
**Pull request by lvzh90**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Map pipelineParams = [ "mavenAdditionalArgs" : "-U -Djava.net.preferIPv4Stack=true -Dsurefire.rerunFailingTestsCount=5 -Dmaven.test.failure.ignore=true" ]
+
+runtimeExtensionsBuild(pipelineParams)
```
Comment (aiannucci) [inline Jenkinsfile]: Use runtimeProjectsBuild

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Map pipelineParams = [ "mavenAdditionalArgs" : "-U -Djava.net.preferIPv4Stack=true -Dsurefire.rerunFailingTestsCount=5 -Dmaven.test.failure.ignore=true" ]
```
Comment (aiannucci) [inline Jenkinsfile]: It probably works without any of these additional arguments

**Review by aiannucci — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (anosenzo) [reply Jenkinsfile]: maven.test.failure.ignore=true this one sure that is not needed as it is added by default in the mvn() step

File: Jenkinsfile (hunk 1 above)
Comment (lvzh90) [reply Jenkinsfile]: Done

File: Jenkinsfile (hunk 2 above)
Comment (lvzh90) [reply Jenkinsfile]: Run the tests without any of these additional arguments and it was successful

**Review by aiannucci — APPROVED** [review]

**Pull request merged**